recipes/pylibjpeg-rle/recipe.yaml

```
@@ -0,0 +1,50 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+schema_version: 1
+
+context:
+  name: pylibjpeg-rle
+  python_name: pylibjpeg_rle
+  version: 2.1.0
+  python_min: "3.10"
+
+package:
+  name: ${{ name }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/${{ python_name }}-${{ version }}.tar.gz
+  sha256: 3ccba7b912f5bc0facc0f2738a5a55dd843653d2f3c02cc31a09fb5efad4b9c7
+
+build:
+  number: 0
+  script: ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  build:
+    - cargo-bundle-licenses
+    - ${{ compiler('rust') }}
+    - ${{ stdlib("c") }}
+  host:
+    - python ${{ python_min }}.*
```

```suggestion
    - python
```
This is a native Python package so we will be building for all supported versions of Python, not just the minimum supported version.  We can instead just skip Python 3.10

---

See my suggestion further down - there is indeed a better way to skip Python 3.9.  I guess the logic here is more that we have to explicitly tell `rattler-build` what versions from the default set to skip, rather than setting a minimum.